sirepo/api.py

```diff
@@ -0,0 +1,15 @@
+class APIRequest:
+    def __init__(self):
+        return
+
+    def call_api(self, name, kwargs=None, data=None):
+        return uri_router.call_api(name, kwargs=kwargs, data=data)
+
+
+APIBase = APIRequest
```

Comment:
This was a quick fix. Please change all sirepo.api.APIBase to sirepo.request.Base and remove this line. I tested this so it should work:

```sh
cd ~/src/radiasoft/sirepo/sirepo
perl  -pi -e 's{(?<=sirepo.)api.APIRequest}{request.Base}; s{(?<=import sirepo.)api$}{request}; s{(?<=class )API\b}{Request}' *.py */*.py
```

Run this on a committed so that you can verify/undo these changes.

Reply:
3ccfbc6e956420f7da6fc23176cdebb7ae57cfc9